dask/bytes/core.py

```diff
@@ -10,15 +13,15 @@
 
 
 def read_bytes(
-    urlpath,
-    delimiter=None,
-    not_zero=False,
-    blocksize="128 MiB",
-    sample="10 kiB",
-    compression=None,
-    include_path=False,
-    **kwargs,
-):
+    urlpath: str | list[str],
+    delimiter: Any | None = None,
```

Comment:
Any reason not to use `bytes | None` here, per the docstring? Doesn't seem to cause any problems for me locally

Reply:
makes sense!